packages/fast-breakpoint-tracker-react/README.md

```diff
@@ -1,25 +1,30 @@
 # React Breakpoint Tracker
-`fast-breakpoint-tracker-react` is a React component for managing viewport breakpoints. Given a set of breakpoints, this component will monitor the width of the browser window and track the current active breakpoint.
+
+A library for managing viewport breakpoints. Given a set of breakpoints, this component will monitor the width of the browser window and track the current active breakpoint.
 
 ## Installation
+
 `npm i --save @microsoft/fast-breakpoint-tracker-react`
 
 ## Breakpoints
+
 `Breakpoints` are expected as an array. Six default breakpoint values are provided for convenience:
 
 ```jsx
 const defaultBreakpoints = [0, 540, 768, 1084, 1400, 1779];
 ```
 
-### Set custom breakpoint values
-Custom breakpoint values can be set by setting the `breakpoints` value on the BreakpointTracker class with new key:value pairs. Custom keys are supported, but the values must be numbers.
+### Setting custom breakpoint values
+
+Custom breakpoint values are defined by setting the `breakpoints` value on the BreakpointTracker class with new key:value pairs. Custom keys are supported, but the values must be numbers:
```

Comment:
@chrisdholt what does "custom keys are supported" mean here? I think we should add an example of custom keys or remove this language - as is this last sentence doesn't seem pertinent.

Reply:
Was not my change but fixed.